rs/nns/governance/proto/ic_nns_governance/pb/storage.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
```

Comment:
As mentioned, I'm not seeing clearly the difference between this and the internal proto file.

Reply:
Discussed offline - this is to make it simpler to segment internal storage types.  updating it to be pub(crate)